### ISSUE TITLE
Add the side panel to send events, watch the state and actors

### ISSUE THREAD

**Issue by raulconnect** [mattpocock/xstate-vscode#29]

**Comment by mattpocock** [comment]
This is when visualising a machine?

**Comment by raulconnect** [comment]
Yes, the same right panel you have available when working on the visualizer, I need to send events, watch the context and all that stuff I need to have full interaction with the machine... and btw it would be nice if there is a way to save a list of common events so we can replay them easily

**Comment by dsmurl** [comment]
Maybe I'm confused, but isn't this the whole point of the visualizer?  I'm surprised to see this wasn't a MVP feature of this plugin.  There is a simulation mode where it's not an option to send events?  That's the single point of simulating the machine?  Which is the point of having a plugin. 

This plugin has very little value if any without this.  There must be a way to send events with params to test the guards and flow while watching the context.  Watching the context is the core of a state machines guiding flow.  The UI in the plugin has the option to click through events but if there is a guard, like there should be in any significant machine, then the click just chooses an arbitrary guard to execute and the paths other than that guard are not available in the simulation.  

If these are options in the web page version, that doesn't help because you would have to constantly have to cut and paste between the IDE and the web page as the code is updating.  

Can someone tell me what this plugin is for because, it might be me, but someone here is complete confused.  This issue is an MVP that should have been required before beta release yet the issue is 3 years old with no updates.
